lib/internal/webidl.js

```diff
@@ -275,20 +277,34 @@ function createSequenceConverter(converter) {
       const val = converter(res.value, {
         __proto__: null,
         ...opts,
-        context: `${opts.context}, index ${array.length}`,
```

Comment:
This error message change is not semver-major nor minor, as there is only one place using this converter and it's not using the `context`, i.e. the error message is not even displayable in the userland.